CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 # CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
 #
 
-CMAKE_MINIMUM_REQUIRED(VERSION 2.6)
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8.8)
 SET(PROJECTNAME rFSM)
```

Comment:
It was not your choice originally, but I thnk that calling the `CMake` package `rFSM` is misleading, as the `rFSM` project is something else (see https://github.com/kmarkus/rFSM) and this project is some additional tools for `rFSM` . Perhaps it could make sense to call the CMake package `rFSMTools` ?

Reply:
LFTM project name changed into `rFSMTools` (db2ae50)